### DISCUSSION TITLE
github.com/hbstack/docs/modules/sticky-till-the-end/v0.1.0

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#765] [General]
# github.com/hbstack/docs/modules/sticky-till-the-end/v0.1.0

0.1.0 (2024-04-02) Features ✨ add the sticky-till-the-end module (#617) (5c9255d) 

https://hbstack.dev/releases/github.com-hbstack-docs/modules-sticky-till-the-end-v0.1.0/

**Comment by LITUATUI** [comment]
Thanks!

**Reply by razonyang** [reply]
My pleasure.
